package/MDAnalysis/coordinates/chain.py

```diff
@@ -80,68 +166,69 @@ class ChainReader(base.ProtoReader):
+        treat several parts of a long simulation as a single trajectory.
 
         Parameters
         ----------
         filenames : str or list or sequence
-           file name or list of file names; the reader will open all file names
-           and provide frames in the order of trajectories from the list. Each
-           trajectory must contain the same number of atoms in the same order
-           (i.e. they all must belong to the same topology). The trajectory
-           format is deduced from the extension of each file name.
-
-           Extension: `filenames` are either a single file name or list of file
-           names in either plain file names format or ``(filename, format)``
-           tuple combination. This allows explicit setting of the format for
-           each individual trajectory file.
+            file name or list of file names; the reader will open all file names
+            and provide frames in the order of trajectories from the list. Each
+            trajectory must contain the same number of atoms in the same order
+            (i.e. they all must belong to the same topology). The trajectory
+            format is deduced from the extension of each file name.
+
+            Extension: `filenames` are either a single file name or list of file
+            names in either plain file names format or ``(filename, format)``
+            tuple combination. This allows explicit setting of the format for
+            each individual trajectory file.
         skip : int (optional)
-           skip step (also passed on to the individual trajectory readers);
-           must be same for all trajectories
+            skip step (also passed on to the individual trajectory readers);
+            must be same for all trajectories
         dt : float (optional)
-          Passed to individual trajectory readers to enforce a common time
-          difference between frames, in MDAnalysis time units. If not set, each
-          reader's `dt` will be used (either inferred from the trajectory
-          files, or set to the reader's default) when reporting frame times;
-          note that this might lead an inconsistent time difference between
-          frames.
+            Passed to individual trajectory readers to enforce a common time
+            difference between frames, in MDAnalysis time units. If not set, each
+            reader's `dt` will be used (either inferred from the trajectory
+            files, or set to the reader's default) when reporting frame times;
+            note that this might lead an inconsistent time difference between
+            frames.
+        continuous : bool (optional)
+            treat all trajectories as one single long trajectory. Adds several
+            checks. Might not work with all trajectory formats! This should
+            ensure that there is no need to concatenate trajectories of GROMACS
+            simulations
```

Comment:
Add section `Raises` and say what is raised when not all trajectories are of the same format or if they are of a format that is not supported.

Reply:
`This section should be used judiciously, i.e., only for errors that are non-obvious or have a large chance of getting raised.`

This is from the numpy guidelines. The error messages raised are pretty clear.